srcpkgs/Quake3e/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'Quake3e'
+pkgname=Quake3e
+version=2021.03.28
+revision=1
+_version=${version//./-}
+wrksrc=Quake3e-$_version
+
+case $XBPS_TARGET_MACHINE in
+	x86_64*) _arch=x86_64 ;;
+	i686*) _arch=x86 ;;
+	armv7l*) _arch=arm ;;
+	aarch64*) _arch=aarch64 ;;
+	*) _arch=$XBPS_TARGET_MACHINE ;;
+esac
+
+build_style=gnu-makefile
+make_build_args="ARCH=$_arch USE_SYSTEM_JPEG=1 USE_RENDERER_DLOPEN=0 USE_CURL_DLOPEN=0 USE_SDL=0"
+make_use_env=true
+makedepends="libcurl-devel libglvnd-devel libXxf86dga-devel alsa-lib-devel libXrandr-devel libXxf86vm-devel libjpeg-turbo-devel"
+depends="libglvnd libXxf86dga alsa-lib libXrandr libXxf86vm"
```

Comment:
`depends` isn't needed, it should be detected automatically.

Reply:
These specifically are `dlopen()`ed at runtime.